src/components/FilmDetails/FilmDetails.js

```diff
@@ -1,15 +1,28 @@
 /* eslint-disable max-len */
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { store } from '../../store';
 
 export class FilmDetails extends Component {
+  componentDidMount() {
+    this.unsubscribe = store.subscribe(() => {
+      this.forceUpdate();
```

Comment:
Do not use forceUpdate, setState instead

Reply:
> Do not use forceUpdate, setState instead

fixed